docs/low-code-spark/extensibility/gem-builder.md

```diff
@@ -6,20 +6,857 @@ description: Gem-Builder
 tags: []
 ---
 
-Prophecy Data Engineering system can be extended with **Gem Builder** to add **your own standard sources, targets and
-transforms** to Spark and roll it out to your entire team. You can build **your own frameworks with** custom Data Quality
-library, or auditing library.
+:::caution Enterprise Only
+
+Please [contact us](https://www.prophecy.io/request-a-demo) to learn more about the Enterprise offering.
+
+:::
+
+[Gems](docs/concepts/gems.md) are operations you can add to your Pipeline to perform actions on your data. The **Gem Builder** allows you to create custom UI experiences that will generate optimized code wherever it's used. You can add your own custom sources, targets, and transformations and roll them out to your entire team. You can even build a custom Data Quality library, or an auditing library.
 
```

Comment:
Each Prophecy Pipeline is composed of individual operations, or [Gems](docs/concepts/gems.md), that perform actions on data. While Prophecy offers dozens of Gems out-of-the-box, some data practitioners want to extend this idea and create their own Gems. Luckily, **Gem Builder** allows enterprise users to add custom Gems. Create custom source, target, and transformation Gems, publish, and your team can utilize your custom Gem. Like all Prophecy Gems, the Custom Gem code will be saved to git using Python or Scala. 

Consider repeated patterns when deciding when to create a custom Gem. Does your team use a particular business logic frequently? Do you notice yourself doing the same Data Quality checks more than once a month? Does your team have an auditing use case that could benefit from re-usable code? Consider a custom Gem to capture these logical patterns for re-use!